app/src/forms/form/formMetadata/routes.js

```diff
@@ -0,0 +1,30 @@
+const routes = require('express').Router();
```

Comment:
We do need OpenAPI docs for all new routes - they're in `app/src/docs/v1.api-spec.yaml`. I think maybe the form object in the docs will need to be updated too?